python/lsst/ts/GenericCamera/__init__.py

```diff
@@ -23,6 +23,7 @@
     from .version import *
 except ModuleNotFoundError:
     __version__ = "?"
+
```

Comment:
Please consider `from .config_schema import *` here so that you can write a test_validation.py unit test (which I hope you will do -- not necessarily in this ticket -- it's a good thing to have for any configurable CSC).

Reply:
Thanks for catching that. I have added the line. The `test_validation.py` suggestion is new to me and I did not implement that in this or any other CSC related to this ticket.

Reply:
See ts_ATDome for an example of test_validation (though I need to change setUp to set to `self.schema = ATDome.CONFIG_SCHEMA`, replacing many lines of code in that method.